docs-linting/markdownlint-custom-rules/codesnippet.js

```diff
@@ -54,16 +79,6 @@ module.exports = {
 									});
 								}
 
-								//do we have language?
```

Comment:
Can we delete this?

Reply:
We dont need this. The requirement for the linting is not to require language.